content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -605,6 +605,13 @@ objects in your Kubernetes deployment, you MUST use this admission controller to
 
 See the [resourceQuota design doc](https://git.k8s.io/community/contributors/design-proposals/resource-management/admission_control_resource_quota.md) and the [example of Resource Quota](/docs/concepts/policy/resource-quotas/) for more details.
 
+### RuntimeClass {#runtimeclass} {{< feature-state for_k8s_version="v1.16" state="alpha" >}}
```

Comment:
This feature has entered Beta since 1.14?

Reply:
This is a little confusing.
RuntimeClass in `Admission Controller` is started 1.16.
RuntimeClass in `Feature Gates` is started 1.12 (beta at 1.14)

Reply:
@makocchi-git 👋 Will you please link an issue or PR from k/k showing the feature entering alpha in 1.16?

Reply:
@zacharysarah I think this [PR](https://github.com/kubernetes/kubernetes/pull/81072) is what you want.